docs/README.md

```diff
@@ -17,23 +17,27 @@ Create a cluster with EKS Anywhere and set and export with KUBECONFIG.
 
         make install
 
-1. Run the controller locally:
+2. Run the controller locally:
```

Comment:
It will be numbered automatically when it's rendered, but the source needn't be for that to happen. A lot of editors have a plugin for markdown that will do the renumbering, but that's a separate (and technically unneeded) thing.

Reply:
Yep my editor change this on its own. I ended up editing it in vim so should be good now